llvm/lib/Target/SBF/SBFInstrInfo.h

```diff
@@ -58,9 +58,11 @@ class SBFInstrInfo : public SBFGenInstrInfo {
                         MachineBasicBlock *FBB, ArrayRef<MachineOperand> Cond,
                         const DebugLoc &DL,
                         int *BytesAdded = nullptr) const override;
+  void setHasExplicitSext(bool HasExplicitSext);
+
 private:
   void expandMEMCPY(MachineBasicBlock::iterator) const;
-
+  bool HasExpliciSext;
```

Comment:
This works as is, but maybe it would be better for consistency to name this field HasExplicitSignExt?
```suggestion
  bool HasExplicitSignExt;
```

Reply:
I've fixed that in my latest commit. This comment dates from 6 hours ago. Somehow it was only published now.